src/dsa/rpo_falcon512/math/mod.rs

```diff
@@ -31,6 +34,9 @@ use self::samplerz::sampler_z;
 mod polynomial;
 pub use polynomial::Polynomial;
 
+const MAX_SMALL_POLY_COEFFICENT_SIZE: i16 = (1 << (WIDTH_SMALL_POLY_COEFFICIENT - 1)) - 1;
+const MAX_BIG_POLY_COEFFICENT_SIZE: i16 = (1 << (WIDTH_BIG_POLY_COEFFICIENT - 1)) - 1;
```

Comment:
Noe thing to confirm: the numbers we want to set `MAX_SMALL_POLY_COEFFICENT_SIZE` and `MAX_BIG_POLY_COEFFICENT_SIZE` are 31 and 127 respectively, right?

Reply:
Yes, this is specified explicitly in the [reference](https://falcon-sign.info/falcon.pdf) doc on page 49 for $f$ and $g$ (i.e., 31) and can be computed for $F$ (and $G$) as $2^{8 - 1} - 1$ where $8$ comes from the same page.